awx/main/models/projects.py

```diff
@@ -568,10 +580,10 @@ def preferred_instance_groups(self):
     '''
     def _get_active_jobs(self):
         return UnifiedJob.objects.non_polymorphic().filter(
-            Q(status__in=ACTIVE_STATES) &
+            models.Q(status__in=ACTIVE_STATES) &
             (
-                Q(Job___project=self) |
-                Q(ProjectUpdate___project=self)
+                models.Q(Job___project=self.project) |
+                models.Q(ProjectUpdate___project=self.project)
```

Comment:
This is not correct because it returns all project updates for the project, if it's called on a project update, it should only return itself at most

really, it should never have been defined on the project update model in the first place